### PR TITLE
Fixed code style consistency more

### PR DESCRIPTION
**Pull request by sjfricke**
As someone who has looked and used all these tutorials, I noticed a small trend of deviation from the consistent styling from the first few tutorials. Since this might be someone else first time learning how to write android Vulkan code I feel a consistency styling makes the code more readable.

Ran all tutorials to confirm still build and run

### CODE REVIEW

File: tutorial04_first_window/app/src/main/jni/VulkanMain.cpp [hunk 1]
```diff
@@ -207,6 +207,17 @@ void CreateSwapChain() {
   LOGI("<-createSwapChain");
 }
 
+void DeleteSwapChain(void) {
```
Comment (sjfricke) [inline tutorial04_first_window/app/src/main/jni/VulkanMain.cpp]: The `DeleteSwapChain` under the `CreateSwapChain` as all the other "Create..." have the "Delete..." under it

File: tutorial05_triangle/app/src/main/jni/VulkanMain.cpp [hunk 2]
```diff
@@ -64,14 +64,14 @@ struct VulkanSwapchainInfo {
 VulkanSwapchainInfo  swapchain;
 
 struct VulkanBufferInfo {
-    VkBuffer vertexBuf;
+    VkBuffer vertexBuf_;
```
Comment (sjfricke) [inline tutorial05_triangle/app/src/main/jni/VulkanMain.cpp]: keeping the idea of these Vulkan struct groups ending with an underscore

**Comment by ggfan** [comment]
LGTM.  could you save your code locally, then try with `clang-format -i -style Google`, compare your changes against clang-format changes?  so we get idea how different it is ( this code here was not run with clang-formt at the beginning though, so I am leaning toward to merge your change ).  Thanks

**Comment by sjfricke** [comment]
Wow, I can't believe I didn't know about clang-format, that gonna save me so much time in my life in the future

**Comment by ggfan** [comment]
thanks for PR. merging!

**Pull request merged**